src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -324,4 +333,47 @@ export class MongoEntityConverter {
   public convertAdLibs(mongoAdLibPieces: MongoAdLibPiece[]): AdLibPiece[] {
     return mongoAdLibPieces.map((piece) => this.convertAdLib(piece))
   }
+
+  public convertStudio(mongoStudio: MongoStudio): Studio {
+    const defaultNumberOfObjects: number = 1
+    const defaultLookAheadDistance: number = 10
+    const layers: StudioLayer[] = []
+    for (const mapping in mongoStudio.mappings) {
+      layers.push({
+        name: mapping,
+        lookAheadMode: this.mapLookAheadNumberToEnum(mongoStudio.mappings[mapping].lookahead),
```

Comment:
After searching a bit it does seem like `lookahead` is the right way to reference this, i know this might be a bit late to bring it up but it looks weird to me :-D